test/testfiles.jl

```diff
@@ -1,4 +1,10 @@
 function testfile(file, kwargs, expected_sz, expected_sch, testfunc; dir=dir)
+    kwargs = if :ignoreemptylines ∉ kwargs
+        (ignoreemptylines=false, kwargs...)
```

Comment:
Hmmm, this is the only case that makes me a little uncomfortable. I know it's a bit of a pain, but I'd rather go through all the test files and add `ignoreemptylines=false` manually if necessary; my guess is it won't affect the majority of cases.